Dynatrace/dynatrace-operator
# Generate mock for webhook.PodMutator interface using vectra/mockery

## Description

Removed old manual mock impl and added the auto generated one via Mockery.

## How can this be tested?

`make go/gen_mocks` to generate mocks
`make go/test` to see everything is working fine

## Checklist
